conf/main/logback.xml

```diff
@@ -62,11 +62,11 @@
         <appender-ref ref="${jetty.log.appender:-MAIN}"/>
     </logger>
 
-    <logger name="ai.grakn" level="${grakn.log.level}">
+    <logger name="ai.grakn" level="${grakn.log.level:-INFO}">
```

Comment:
that is just a default argument in case it is not provided in the config file - this is the situation when starting using the migrator script - by default it was using DEBUG